pages/layanan/kependudukan-dan-tempat-tinggal/_slug/index.vue

```
@@ -32,14 +39,30 @@
 
 <script>
 export default {
+  async asyncData ({ params, $axios, error }) {
+    const slug = await params.slug
+
+    try {
+      const response = await $axios.get(`/v1/public/public-service/slug/${slug}`)
+
+      const { data: service } = response.data
+
+      const jumbotron = {
+        title: service.name,
+        subtitle: service.description,
+        backgroundImageUrl: '/images/jumbotron/default.webp'
+      }
+
+      return { service, jumbotron }
+    } catch (e) {
+      // silent error
```

### Ask

Hello @naufalihsank, I wonder, what is the meaning in comment `silent error` ?

---

we don't do anything 🤣

cause i already handle the error by setting the default value for `service` and `jumbotron`

or maybe we should do something in this case of error @Ibwedagama ?

---

my initial plan is to show some error message or toast to the user, but to do so we need an error component/alert specific for this purpose. at that time I don't know what to do and write `//silent error` instead 🤣

I've asked UID for providing the design, but I think we forgot about it 🤣

cc @naufalihsank @doohanas

---

in this scenario, let say user write the wrong URL, we could redirect them to an error 404 page using this syntax:

```js
catch (e) {
  error({ statusCode: e.response.status, message: e.response?.data?.message })
}

```

cc @naufalihsank

---

Okay, thanks for the answer.  @Ibwedagama @naufalihsank

---

ooh, so that's the purpose of that syntax, thanks for the info

i implemented that 404 redirect in [here](https://github.com/jabardigitalservice/portal-jabar-frontend/pull/357/commits/b09d26cca437f3190689a08b9e2e3915605daba4)